### PR TITLE
chore: update npm dependencies

### PR DESCRIPTION
**Pull request by AnastasiaArcadia**
**Related to:** https://oat-sa.atlassian.net/browse/AUT-192

**How to test**
1. Log in as Admin in Chrome or Internet Explorer
2. Choose 'Items' tab
3. Choose item and click 'Authoring' button
4. Choose 'Response' tab of interaction and set choice sequence that is different from order in which choices are listed in 'Question' tab
5. Choose 'Question' tab and observe choice sequence
6. Choice options should preserve initial order in which they are created upon adding Order Interaction onto canvas

Env: http://test-anastasia-2.playground.kitchen.it.taocloud.org:45401/

### CODE REVIEW

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: views/package.json [hunk 1]
```diff
@@ -9,6 +9,6 @@
     },
     "dependencies": {
         "@oat-sa/tao-item-runner": "0.7.1",
-        "@oat-sa/tao-item-runner-qti": "0.21.10"
+        "@oat-sa/tao-item-runner-qti": "git://github.com/oat-sa/tao-item-runner-qti-fe#fix/AUT-192/wrong-choices-order"
```
Comment (404passionFruit) [inline views/package.json:12]: Just a reminder to replace this with proper version before merge

**Review by 404passionFruit — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`
- [x] Pull request title and description are meaningful
- [x] Commits are following conventional commits
- [x] Kitchen links are not exposed in the description

**Review by habemuscode — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [ ]  The feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request's target is not master
- [x]  Pull request title and description are meaningful
- [x]  Commits are following conventional commits
- [x]  Kitchen links are not exposed in the description

**Comment by AntonNeviarouski** [comment]
QA verified on http://test-anastasia-2.playground.kitchen.it.taocloud.org:45401/

**Pull request merged**